test/functional/fixtures/regression/gh-423/pages/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>GH-423</title>
+</head>
+<body>
+<div id="target1" style="background: green;position: absolute; width:100px; height:100px;">
+</div>
+<div id="element" style="top:200px;background: red;position: absolute; width:100px; height:100px;">
+</div>
+<div id="target2" style="left: 150px;background: silver;position: absolute; width:100px; height:100px;">
+</div>
+<div id="child" style="left: 150px;top:200px;background: blue;position: absolute; width:100px; height:100px;">
+</div>
+<div id="target3" style="left: 300px;background: violet;position: absolute; width:50px; height:50px;">
+</div>
+<div id="newParent" style="left: 300px;top:200px;background: yellow;position: absolute; width:100px; height:100px;">
+</div>
+<div id="target4" style="left:450px; background:orange;position: absolute; width:100px; height:100px;">
+</div>
+<input id="input" style="left:450px; top: 200px; position: absolute;">
+<script>
+    function saveEventToStorage (e) {
+        window.eventStorage.push(e.currentTarget.id + ' ' + e.type);
```

Comment:
it's reasonable. @georgiy-abbasov is right. In our case it's better to know what the handlers were called. So, I think we should leave it as is